### ISSUE TITLE
HDF format: use pytables internally instead of pandas

### ISSUE THREAD

**Issue by alixdamman** [larray-project/larray#724]
See discussion from issue #563

**Comment by alixdamman** [comment]
@gdementen have you already started something ? 
I guess the idea is to have our own HDFStore, probably somewhat based on the one implemented in pandas ?

**Comment by gdementen** [comment]
Nope, didn't start something. In the long term, yes, but I meant first to only bypass pandas when saving/loading hdf stuff. This shouldn't be hard though.
